packages/design-system/src/components/optimized-image/utils/validate-srcset.ts

```diff
@@ -0,0 +1,27 @@
+import { DesignSystemError } from '../../../utils/design-system-error';
+
+/** @internal */
+export const widthRegexp = /^((\s*\d+w\s*(,|$)){1,})$/;
+
+/** @internal */
+export const densityRegexp = /^((\s*\d+(\.\d+)?x\s*(,|$)){1,})$/;
+
+/** @internal */
+export const validateSrcset = (src: string, srcset?: string | null) => {
```

Comment:
This function doesn't return anything unlike your other validators. Remove _useless_ returns there or add it here.